src/avram/migrator/alter_table_statement.cr

```diff
@@ -175,12 +175,25 @@ class Avram::Migrator::AlterTableStatement
     @fill_existing_with_statements << "ALTER TABLE #{@table_name} ALTER COLUMN #{column} SET NOT NULL;" unless nilable
   end
 
-  {% symbol_expected_message = "%s expected a symbol like ':user', instead got: '%s'" %}
+  macro symbol_expected_error(action, name)
+
+    {% raise <<-ERROR
+
+    #{action} expected a symbol like ':user', instead got: '#{name}'.
```

Comment:
If we have a check for it being a `TypeDeclaration`, then we can say `like ':{{ name.var }}'`.

Reply:
Hi Jeremy - done the change but see my earlier comment about Git. Cheers.